app/controllers/spree/admin/payment_methods_controller.rb

```diff
@@ -100,6 +100,10 @@ def show_provider_preferences
 
       private
 
+      def payment_method_params
+        params.require(:payment_method).permit!
```

Comment:
Again, I don't think we should be using `#permit!` unless there's absolutely no other option...

Reply:
this is coming from spree. I'll improve this :+1:

Reply:
done by listing the params.
I also fixed the respective spec.